.github/workflows/ci.yml

```diff
@@ -117,3 +118,85 @@ jobs:
         run: |
           cd cmd
           ../bin/goreleaser --snapshot
+
+  release:
+    name: Relase
+    runs-on: ubuntu-latest
+    needs: [lint, test]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/v')
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+
+      - name: Check out the code
+        uses: actions/checkout@v1
+
+      - name: Export version from tag name
+        run: |
+          echo "NIMONA_VERSION=$(git describe)" >> $GITHUB_ENV
```

Comment:
Owh that’s pretty cool

Reply:
I hope it works. Never used it before, so this is purely based on the GitHub Actions docs :P